.github/workflows/release.yaml

```diff
@@ -261,6 +234,7 @@ jobs:
         run: |
           npm install -g @e2b/cli@latest
       - name: Publishing Template
+        if: github.event_name == 'release'
```

Comment:
The release workflow now only publishes packages when the event name is 'release'.  Previously, it may have published on other events, potentially leading to unintended deployments to PyPI.  To revert to the previous behavior (publishing regardless of event), remove the `if: github.event_name == 'release'` conditions added in lines 41, 75, 100, 133, 142, 149, 158, 165, 174, 181, 190, 218, 237.